integration/spark/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -1198,10 +1172,16 @@ case class LoadTableUsingKettle(
         GlobalDictionaryUtil
           .generateGlobalDictionary(sqlContext, carbonLoadModel, relation.tableMeta.storePath,
             dataFrame)
-        CarbonDataRDDFactory
-          .loadCarbonData(sqlContext, carbonLoadModel, storeLocation, relation.tableMeta.storePath,
+        CarbonDataRDDFactory.loadCarbonData(sqlContext,
+            carbonLoadModel,
+            storeLocation,
+            relation.tableMeta.storePath,
```

Comment:
Actually one is temporary location to store in the local disk of node. But it is not required as it got added later. So I am removing it.